nx.json

```diff
@@ -14,19 +14,21 @@
     }
   ],
   "release": {
-    "projects": ["*"],
+    "projects": [
+      "*"
+    ],
     "changelog": {
       "projectChangelogs": true,
       "workspaceChangelog": {
         "createRelease": "github",
         "file": "CHANGELOG.md"
+      },
+      "git": {
+        "commit": true,
+        "tag": true,
+        "commitMessage": "chore(release): {version} [skip ci]"
       }
     },
-    "git": {
-      "commit": true,
-      "tag": true,
-      "commitMessage": "chore(release): {version} [skip ci]"
-    },
```

Comment:
The documentations place the `"git"` node under `"release"`: https://nx.dev/reference/nx-json#git

This should probably be reverted?